config/after_symlink_shared.yml

```diff
@@ -1,17 +1,41 @@
+- import_tasks: "{{ ansistrano_symfony_before_composer_file | default('steps/empty.yml') }}"
```

Comment:
Here it should use `include_tasks` to be able to use runtime variables in any ansistrano_symfony** variable.